misc/assets/inforequest-flow.svg

```diff
@@ -0,0 +1,343 @@
+  <path d="M 750,100 v 1125" fill="none" stroke="black" stroke-width="1pt" stroke-dasharray="2pt" />
+
+  <g transform="translate(200 120)">
+    <text class="title">Applicant</text>
+  </g>
+
+  <g transform="translate(550 120)">
+    <text class="title">Obligee</text>
+  </g>
+
+  <g transform="translate(950 120)">
+    <text class="title">Advanced Obligee</text>
+  </g>
+
+  <g transform="translate(200 220)">
+    <path class="arrow" d="M 0,-70 v 40" marker-start="url(#start)" />
+    <use href="#action" />
+    <foreignObject x="-100" y="-30" width="200" height="60">
+      <xhtml:div class="action">
+        <xhtml:div class="caption">Request</xhtml:div>
+        <xhtml:div class="note">obligee: 8 WD + SZ</xhtml:div>
+        <xhtml:div class="note">after: DD or SD + 1 WD</xhtml:div>
+      </xhtml:div>
+    </foreignObject>
+    <path class="arrow" d="M 100,0 h 380 l 20,20" />
+  </g>
+
+  <g transform="translate(950 220)">
+    <path class="arrow" d="M 0,-70 v 40" marker-start="url(#start)" />
+    <use href="#action" />
+    <foreignObject x="-100" y="-30" width="200" height="60">
+      <xhtml:div class="action">
+        <xhtml:div class="caption">Adv. Request</xhtml:div>
+        <xhtml:div class="note">obligee: 8 WD + SZ</xhtml:div>
+        <xhtml:div class="note">after: DD or LD + 4 WD</xhtml:div>
+      </xhtml:div>
+    </foreignObject>
+    <path class="arrow" d="M -100,0 h -130 l -20,20" />
+  </g>
+
+  <g transform="translate(550 320)">
+    <path class="arrow" d="M 150,0 h -50" />
+    <use href="#action" />
+    <foreignObject x="-100" y="-30" width="200" height="60">
+      <xhtml:div class="action">
+        <xhtml:div class="caption">Confirmation</xhtml:div>
+        <xhtml:div class="note">obligee: PA DL + PA SZ</xhtml:div>
+        <xhtml:div class="note">after: PA after</xhtml:div>
+      </xhtml:div>
+    </foreignObject>
+    <path class="arrow" d="M 0,-30 v -20 h 130 l 20,20" />
+  </g>
+
+  <g transform="translate(550 470)">
+    <path class="arrow" d="M 150,0 h -50" />
+    <use href="#action" />
+    <foreignObject x="-100" y="-30" width="200" height="60">
+      <xhtml:div class="action">
+        <xhtml:div class="caption">Clarif. Req.</xhtml:div>
+        <xhtml:div class="note">applicant: 7 CD no SZ</xhtml:div>
+        <xhtml:div class="note">after: DD</xhtml:div>
+      </xhtml:div>
+    </foreignObject>
+    <path class="arrow" d="M -100,0 h -50 v -50 h 150 v 20" />
+    <path class="arrow" d="M -100,0 h -250 v -45" />
+    <g transform="translate(-330 5)">
+      <text class="label">
+        <tspan>at most 3 CD</tspan>
+        <tspan x="0" dy="1.2em">after DL</tspan>
+      </text>
+    </g>
+  </g>
+
+  <g transform="translate(200 395)">
+    <use href="#action" />
+    <foreignObject x="-100" y="-30" width="200" height="60">
+      <xhtml:div class="action">
+        <xhtml:div class="caption">Clarif. Resp.</xhtml:div>
```

Comment:
Poradie Action `Clarif. Req.` predbieha action `Clarif. Resp.` Tiez poradie jednotlivych boxov je po stlpcoch, a nie po riadkoch.

Reply:
Tu som do zdrojakov jednotlive stavy daval v poradi v akom vznikaju v casovom zlede, hoc vizualne su to tri stlpce vedla seba.